cluster/storage/src/main/java/com/linkedin/openhouse/cluster/storage/selector/StorageSelectorConfig.java

```diff
@@ -21,7 +22,7 @@ public class StorageSelectorConfig {
 
   @Autowired StorageProperties storageProperties;
 
-  @Autowired List<StorageSelector> storageSelectors;
+  @Lazy @Autowired List<StorageSelector> storageSelectors;
```

Comment:
why was this required to be lazy? `provideStorageSelector` will get called during boottime, its unlikely @lazy would help much.

Reply:
I see why this @lazy might've needed, but I've feeling this will error out. Can we fix the lazy loading on the selector side?